pkg/collector/py/utils_test.go

```diff
@@ -75,6 +75,67 @@ func TestFindSubclassOf(t *testing.T) {
 	assert.Equal(t, 1, sclass.RichCompareBool(barClass, python.Py_EQ))
 }
 
+func TestSubprocessBindings(t *testing.T) {
```

Comment:
could you also write a test that executes a python check that uses `get_subprocess_output`? (a bit like what the tests in `check_test.go` do)